File: misc/enqueue.pl
#!/usr/bin/env perl

use Redis;
use JSON;

use lib qw( lib );

use Pesque;

Pesque::enqueue( 'MOHandler', {
  msisdn => '447760208493',
  body => 'Heya'
} );

package MOHandler;

use Moose;

sub queue {
  'mos'
}

File: misc/enqueue.rb
#!/usr/bin/env ruby

require 'rubygems'
require 'resque'

class MOHandler
  @queue = 'mos'

  def self.perform( params )
    puts "received message from #{params[:msisdn]}"
  end
end

Resque.enqueue( MOHandler, { :msisdn => '447760208493', :body => 'Hai.' } )

